packages/react-native-gesture-handler/src/v3/types.ts

```diff
@@ -34,5 +34,9 @@ export type TouchEvent =
 
 export type CallbackHandlers = Omit<
   HandlerCallbacks<Record<string, unknown>>,
-  'gestureId' | 'handlerTag' | 'isWorklet'
+  | 'gestureId'
+  | 'handlerTag'
+  | 'isWorklet'
+  | 'changeEventCalculator'
+  | 'onChange'
 >;
```

Comment:
Can we get a todo to replace this type with a v3-specific one?

Reply:
Sure, I assumed that we will remember, but it won't hurt to add `todo`.